shared/util/saga.js

```diff
@@ -11,6 +12,27 @@ import {isArray} from 'lodash-es'
 
 export type SagaGenerator<Yield, Actions> = Generator<Yield, void, Actions>
 
+class SagaLogger {
+  error: LogFn
+  warn: LogFn
+  info: LogFn
+  debug: LogFn
+  isTagged = false
+  constructor(actionType, fcnTag) {
+    const prefix = `${fcnTag} [${actionType}]:`
+    this.debug = (...args) => logger.debug(prefix, ...args)
+    this.error = (...args) => logger.error(prefix, ...args)
+    this.info = (...args) => logger.info(prefix, ...args)
+    this.warn = (...args) => logger.warn(prefix, ...args)
+  }
+  // call this first in your saga if you want chainAction / chainGenerator to log
+  // before and after you run
+  tag = () => {
+    this.debug('->')
```

Comment:
This part is `debug` but the other part is `info` on purpose?

Reply:
Nope! Thanks